dcore/configuration.py

```diff
@@ -35,22 +35,22 @@ class OutputParameters(Configuration):
     """
 
     Verbose = False
-    dumpfreq = 10
+    dumpfreq = 1
     dumplist = None
     dirname = None
     #: Should the output fields be interpolated or projected to
     #: a linear space?  Default is interpolation.
     project_fields = False
-    #: Can be a list of fields for computing perturbations.
-    #: Must be in the same order as fieldlist with entry None
-    #: if a particular perturbation field is not required.
-    meanfields = [None, None, None]
+    #: Dump error fields for steady state simulation. Default is False.
+    steady_state_dump_err = {}
+    #: Dictionary of fields for computing perturbations. Default is False.
+    meanfields = {}
```

Comment:
meanfields is now a dictionary - I didn't like the assumption that it should have 3 entries (corresponding to the 3 fields in the compressible model) although it's a little pedantic as shallow water doesn't have mean fields...